### ISSUE TITLE
Wrong export package name for bean-validator-cdi module

### ISSUE THREAD

**Issue by AngryGami** [eclipse-ee4j/glassfish#22712]
## Environment Details
* GlassFish Version (and build number): master (at least to 4013c90), 5.0 release suffer from this as well
* JDK version: 1.8_192, but irrelevant to this problem
* OS: Linux alpine, again, irrelevant to this problem

----------

## Problem Description
Deploying of application fails with:
```
  Exception while loading the app : java.lang.ClassNotFoundException: org.hibernate.validator.cdi.internal.interceptor.ValidationInterceptor
java.lang.ClassNotFoundException: org.hibernate.validator.cdi.internal.interceptor.ValidationInterceptor
        at com.sun.enterprise.loader.ASURLClassLoader.findClassData(ASURLClassLoader.java:841)
        at com.sun.enterprise.loader.ASURLClassLoader.findClass(ASURLClassLoader.java:718)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
...
```

Looking for this class lead me to bean-validator-cdi module which is repackaged version of hibernate-validator-cdi. Though when I look at MANIFEST.MF file of repackaged jar I see following:

```
Export-Package: org.hibernate.validator.internal.cdi.interceptor;version
 ="6.0.2.Final",org.hibernate.validator.internal.cdi;version="6.0.2.Fina
 l"
```
See the problem? No?
And now?:
```
...
org.hibernate.validator.cdi.internal.interceptor.ValidationInterceptor
...
org.hibernate.validator.internal.cdi.interceptor
...
```
It look to me that jar is exporting package that doesn't exist and therefore classloader cannot access it. And indeed when I fixed bean-validator-cdi jar manifest - everything is deployed and worked again. 

Please fix that.

## Steps to reproduce

Deploy application with JPA entities in it

## Impact of Issue

Can't run my application.

**Comment by AngryGami** [comment]
I this so hard to fix?

**Comment by bshannon** [comment]
I don't know.  We're all busy with the Jakarta EE 8 release and haven't had time to look at this.
Sorry.

**Comment by AngryGami** [comment]
https://github.com/eclipse-ee4j/glassfish/pull/22792
made pull request

**Comment by bshannon** [comment]
Fixed by #22798.
